web/app/scripts/views/map/map-partial.html

```diff
@@ -6,7 +6,11 @@
                        max-date="ctl.maxDate"
                        toddow="ctl.toddow">
         </driver-charts>
-        <driver-interventions params="ctl.recordQueryParams"></driver-interventions>
+
+        <driver-interventions
+            ng-if="ctl.userCanWrite" params="ctl.recordQueryParams">
+        </driver-interventions>
+
```

Comment:
Interventions/Secondary Record Type still comes up in the map layer picker. See line 398 in views/map/layers-controller.js